client/core/wallet.go

```diff
@@ -340,15 +340,15 @@ func (w *xcWallet) Connect() error {
 	// ConnectOnce so that the ConnectionMaster's On method will report false.
 	err := w.connector.ConnectOnce(context.Background())
 	if err != nil {
-		return err
+		return fmt.Errorf("%s: %w", unbip(w.AssetID), err)
```

Comment:
Would be `w.Symbol` for these.
However, the caller knows what wallet it's using, so instead of decorating every error returned from this method with a symbol, it would be better to have the caller do it.  AFAICT, that's only `(*Core).connectWalletResumeTrades`, so change `codedError` there to `newError` and it's done.  That's the pattern in the other methods that surround it.

Reply:
I see we currently do `unbip(w.AssetID)` even when we have access to `w.Symbol`, would you want me to change those too to use `w.Symbol`?

Reply:
Just here is fine.  That field happens to be new is all.